tests/pytorch/test_fused_optimizer.py

```diff
@@ -169,6 +171,87 @@ def test_frozen_model(self):
 
             torch.testing.assert_close(ref_param, tst_param)
 
+    def test_bf16_model_weight_cast(self):
```

Comment:
We should skip this test on systems that don't support BF16. See:
https://github.com/NVIDIA/TransformerEngine/blob/c71e5e989ee09078249cfaf5ab27f24db059a329/transformer_engine/pytorch/utils.py#L239

Reply:
Done